metrics/scaling/k8s_scale.sh

```diff
@@ -104,12 +104,17 @@ EOF
 		cpu_idle=${cpu_idle:-0}
 		mem_free=${mem_free:-0}
 
+		# bash arrays don't handle hyphens in the index as it treats
+		# a name like "azure-east-1" as 0-0-1, so remove any hyphens in the
+		# name, will work as long as hyphens aren't the only difference
+		node_index=$(tr -d [=-=] <<< $node)
```

Comment:
I'd probably `s/-/_/` rather than remove them - that way the names are still readable when we debug etc.